src/js/messaging/utils/helpers.js

```diff
@@ -85,7 +85,7 @@ export function formattedDate(date, options = {}) {
   let dateString;
 
   if (momentDate.isSame(now, 'd')) {
-    dateString = momentDate.format('HH:mm');
+    dateString = `${momentDate.format('HH:mm')} <abbr title="Eastern Standard Time">EST</abbr>`;
```

Comment:
Ok, I'll undo this and let you handle that. I am still using a mock API and have no way to test this on my end.